tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/dashboards/Course_Comparison_Dashboard.yaml

```diff
@@ -0,0 +1,680 @@
+description: null
+metadata:
+  chart_configuration:
+    '1273':
+      crossFilters:
+        chartsInScope:
+        - 1274
+        - 1275
+        - 1276
+        - 1277
+        - 1278
+        - 1279
+        - 1292
+        - 1294
+        - 1295
+        scope: global
+      id: 1273
+    '1274':
+      crossFilters:
+        chartsInScope:
+        - 1273
+        - 1275
+        - 1276
+        - 1277
+        - 1278
+        - 1279
+        - 1292
+        - 1294
+        - 1295
+        scope: global
+      id: 1274
+    '1275':
+      crossFilters:
+        chartsInScope:
+        - 1273
+        - 1274
+        - 1276
+        - 1277
+        - 1278
+        - 1279
+        - 1292
+        - 1294
+        - 1295
+        scope: global
+      id: 1275
+    '1276':
+      crossFilters:
+        chartsInScope:
+        - 1273
+        - 1274
+        - 1275
+        - 1277
+        - 1278
+        - 1279
+        - 1292
+        - 1294
+        - 1295
+        scope: global
+      id: 1276
+    '1277':
+      crossFilters:
+        chartsInScope:
+        - 1273
+        - 1274
+        - 1275
+        - 1276
+        - 1278
+        - 1279
+        - 1292
+        - 1294
+        - 1295
+        scope: global
+      id: 1277
+    '1278':
+      crossFilters:
+        chartsInScope:
+        - 1273
+        - 1274
+        - 1275
+        - 1276
+        - 1277
+        - 1279
+        - 1292
+        - 1294
+        - 1295
+        scope: global
+      id: 1278
+    '1279':
+      crossFilters:
+        chartsInScope:
+        - 1273
+        - 1274
+        - 1275
+        - 1276
+        - 1277
+        - 1278
+        - 1292
+        - 1294
+        - 1295
+        scope: global
+      id: 1279
+    '1292':
+      crossFilters:
+        chartsInScope:
+        - 1273
+        - 1274
+        - 1275
+        - 1276
+        - 1277
+        - 1278
+        - 1279
+        - 1294
+        - 1295
+        scope: global
+      id: 1292
+    '1294':
+      crossFilters:
+        chartsInScope:
+        - 1273
+        - 1274
+        - 1275
+        - 1276
+        - 1277
+        - 1278
+        - 1279
+        - 1292
+        - 1295
+        scope: global
+      id: 1294
+    '1295':
+      crossFilters:
+        chartsInScope:
+        - 1273
+        - 1274
+        - 1275
+        - 1276
+        - 1277
+        - 1278
+        - 1279
+        - 1292
+        - 1294
+        scope: global
+      id: 1295
+  color_scheme: supersetColors
+  color_scheme_domain:
+  - '#1FA8C9'
+  - '#454E7C'
+  - '#5AC189'
+  - '#FF7F44'
+  - '#666666'
+  - '#E04355'
+  - '#FCC700'
+  - '#A868B7'
+  - '#3CCCCB'
+  - '#A38F79'
+  - '#8FD3E4'
+  - '#A1A6BD'
+  - '#ACE1C4'
+  - '#FEC0A1'
+  - '#B2B2B2'
+  - '#EFA1AA'
+  - '#FDE380'
+  - '#D3B3DA'
+  - '#9EE5E5'
+  - '#D1C6BC'
+  cross_filters_enabled: true
+  default_filters: '{}'
+  expanded_slices:
+    '1262': true
+    '1267': true
+    '1275': true
+    '1276': true
```

Comment:
I've edited this one on the UI, and wrongly added some charts here that don't belong to this dashboard. Please remove:
- 1262
- 1267